### DISCUSSION TITLE
Yank hinted link without visiting

### DISCUSSION THREAD

**Discussion by agenbite** [qutebrowser/qutebrowser#6773] [Q&A] [answered]
Is it possible to yank a link without visiting it? It'd be equivalent to clicking it with the right button of the mouse and selecting "copy link" in mouse-based browsers.

Thanks for the hard work!

**Comment by toofar** [comment, marked as answer]
Try `;y` (clipboard) or `;Y` (primary selection) as mentioned in the "extended hint mode" part of the cheatsheet https://raw.githubusercontent.com/qutebrowser/qutebrowser/master/doc/img/cheatsheet-big.png

**Reply by agenbite** [reply]
That's great. For some reason I'm having trouble finding these type of questions on my own using Duck Duck Go. Thank you so much!
